.github/workflows/import.yml

```diff
@@ -0,0 +1,31 @@
+name: Import Data from Sirius
+on:
+  schedule:
+    - '0 0 7 ? * MON-FRI *'
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Pull Sirius Source data
+        env:
+          GITHUB_TOKEN:  ${{ secrets.ORG_ACCESS_TOKEN }}
+        run: |
+          mkdir ./source-data
+          gh run download -n data-dictionary-json  -R ministryofjustice/opg-sirius -D ./source-data
+
+      - name: Checkout
+        uses: actions/checkout@v2.3.1
```

Comment:
actions/checkout@v3 is now available to use, also gets rid of your build warnings I believe with the latest https://github.com/ministryofjustice/opg-data-dictionary/actions/runs/3649655918

Reply:
Oh, good shout.